Makefile

```diff
@@ -61,6 +63,10 @@ test: ## Run tests
 	-go clean -testcache &>/dev/null
 	${OPTS} go test ${TEST_OPTS} ./...
 
+test-windows:
+	-go clean -testcache > NUL
+	${OPTS} go test ${TEST_OPTS} ./...
+
 install-linters: ## Install linters
 	# GO111MODULE=off go get -u github.com/FiloSottile/vendorcheck
```

Comment:
`install-linters` is not working because of the comments
```
PS C:\Users\Erson\dmsg> make install-linters
## GO111MODULE=off go get -u github.com/FiloSottile/vendorcheck
process_begin: CreateProcess(NULL, ## GO111MODULE=off go get -u github.com/FiloSottile/vendorcheck, ...) failed.
make (e=2): The system cannot find the file specified.
make: *** [Makefile:71: install-linters] Error 2
```